src/view/channel-detail.js

```diff
@@ -42,6 +42,11 @@ const ChannelDetailView = ({ store, nav }) => (
             {store.selectedChannel.closingTxId}
           </DetailField>
         ) : null}
+        {store.selectedChannel.blocksTilMaturity ? (
+          <DetailField name="Blocks Until Maturity">
```

Comment:
Rather than showing blocks, I think it'd be more intuitive for end users to show days/hours. So we can convert the blocks to days/hours using 144 blocks in a day (6 each hour).

Reply:
As for the copy, maybe: "Time Til Available", "Time Til Recovery"?

Reply:
fixed